dg-backend/app.py

```diff
@@ -15,26 +15,30 @@
     allow_headers=["*"],
 )
 
-@app.post("/rounds", response_model=schemas.RoundInfoResponse)
-def create_round_endpoint(round_info: schemas.RoundInfoCreate, db: Session = Depends(get_db)):
-    return crud.create_round(db, round_info)
-
-@app.get("/rounds", response_model=list[schemas.RoundInfoResponse])
-def get_round_endpoint(db: Session = Depends(get_db)):
-    return [crud.convert_info_response(round) for round in crud.get_rounds(db)]
-
-@app.get("/rounds/{round_id}", response_model=schemas.RoundInfoResponse)
-def get_round_endpoint(round_id: int, db: Session = Depends(get_db)):
-    round_db = crud.get_round(db, round_id)
-    return crud.convert_info_response(round_db)
-
-@app.put("/rounds/{round_id}", response_model=schemas.RoundInfoResponse)
-def update_round_endpoint(round_id: int, round_info: schemas.RoundInfoUpdate, db: Session = Depends(get_db)):
-    return crud.convert_info_response(crud.update_round(db, round_id, round_info))
-
-@app.delete("/rounds/{round_id}", response_model=int)
-def delete_round_endpoint(round_id: int, db: Session = Depends(get_db)):
-    deleted_round_id = crud.delete_round(db, round_id)
-    if deleted_round_id is None:
-        raise HTTPException(status_code=404, detail="Round not found")
-    return deleted_round_id
+@app.post("/rounds/", response_model=schemas.Round)
```

Comment:
trailing slash tripped me up. if we remove it here, it should work both ways (with and without it)
```suggestion
@app.post("/rounds", response_model=schemas.Round)
```